pkg/services/servicebus/plan_schemas.go

```diff
@@ -82,7 +82,11 @@ func generateTopicBindingParamsSchema() service.InputParametersSchema {
 				Description: "Specifies whether to create a subscription in the topic." +
 					" Valid values are [\"yes\", \"no\"]. ",
 				AllowedValues: []string{"yes", "no"},
```

Comment:
You can delete AllowedValues field and adjust the description.

Reply:
Fixed